### DISCUSSION TITLE
關於人工智慧跟天氣預報

### DISCUSSION THREAD

**Discussion by chihlingchu** [1101-GS4719-C/Python-Community-GS4719-C#1258] [General]
相信大家都被這幾天的天氣搞得心煩意亂，一下變冷，一下暴雨，而近百年來因為氣候變遷的關係，「天氣」這件事在人類的世界中已經沒有安寧和平的時候。雖然在這兩方面我都不是專業人士，但我可以粗淺表達一下我的看法。

人工智慧發展提高了天氣預報準確度，這點毋庸置疑，許多靠著傳統數值模擬方法無法做到的，人工智慧都伸出了援手，DeepMind 就是個知名的例子。數值天氣系統 (NWP) 是利用各種物理方程式去生成預測，但是大量計算花費的時間太久，無法作出即時的預報，若是有非線性事件發生，也難以預測，這就是以前大家總是抱怨連連「為什麼天氣預報都不準」的原因。而 DeepMind DGM (Deep Generatine Model of Rain) 則是透過雷達圖去訓練模型，直接預測網格位置的降水機率，而且使用的是過去 20 分鐘前的資料，預測未來約 90 分鐘的降雨，也更有效的模擬非線性的變化，不依賴平流方程。當然，也要感謝氣象雷達的進步，DeepMind 才有機會拿到如此高時間解析度的資料。

以上資訊是從新聞跟一些歷史故事整理出來的，近年來人工智慧在天氣預報上的發展，也是致力於如何讓模型吃最新、最高級的資料，越高品質、越高時間解析度越好，讓他們能分析出最準確的未來，克服氣候變遷在天氣變化上施加的威脅。可是，這些真的取代得了人類智慧本身在天氣預測上的地位嗎？其實當年 DeepMind 技術少了一個重要因素，就是「客觀評估」，當時許多氣候學家對此技術抱持了逆風的態度，他們認為模型產生的數字漂亮又如何？這些數字缺少了人類智慧的判斷，若是哪天缺乏了合理性，別說預報，如果造成政府的公共安全政策出錯，該如何收拾殘局？

大家在看天災片的時候，無論地震還是海嘯，是否都有這麼一號人物出現：天災發生前的研究員，對於電腦預測到的事件驚訝萬分，鏡頭特寫出他們夾雜著驚恐、悲傷，卻又對即將到來的未來感到無奈的表情，我總是對那個表情印象深刻。雖然跟這篇的議題沒什麼關聯，我只是想說「人心」對於任何自然事件的預測，依舊是處在屹立不搖的地位。不管人工智慧的模型如何進步，若是少了專業人士在背後監督預測結果的合理性，人類也只相信電腦的理解，總會有出錯的那一天。

也因此，DeepMind 當年僅僅是抱著「分享」的心態，並沒有實際運用的計畫，雖然沒有得到青睞，但確實也為未來的預測模型改進上提供了不小的助力。而網路上最近有著一個說法：ChatGPT 將會取代天氣預報員？我一樣認為不會，雖然 ChatGPT 對於人心、情緒上的理解正在進步，也可以有效地對很多氣象上的專有名詞做出易懂的解釋，但我相信大家看氣象預報應該還是會想看到漂亮的姊姊為大家講解，而不是一些電腦的背景音。

**Comment by GuanYann** [comment]
透過人工智慧在天氣預報上的發展，可以提高預報的準確度和即時性，克服氣候變遷所帶來的威脅。然而，專業人士在背後的監督和客觀評估仍然至關重要，否則電腦的預測結果可能會缺乏合理性，甚至對公共安全政策造成錯誤影響。在科技進步的同時，人類智慧仍然是必不可少的，尤其在面對自然災害的時候。

**Comment by curtis1017** [comment]
我覺得如果將人工智慧用在天氣預報上不一定會比較精準，主要是因為現在的氣候變遷，常常使我們無法預測未來的天氣，加上他們是由大數據來分析，可是天氣的變化總是有新的。

**Comment by CCello** [comment]
我認為人工智慧在天氣預報方面的發展正在取得重大進展，主要是因為傳統的天氣預報主要依賴於物理模型和數據分析，而人工智慧技術的應用可以進一步提高預報準確性和精度，同時能夠實現更加精細化和個性化的預報。

**Comment by KevinCheng42** [comment]
我同意作者的觀點，天氣是指短時間的氣象變化，像是午後雷陣雨或是各地的降雨機率，這種類型的氣象狀況不容易使用AI或建立模型去預測；相反的，我們如果討論氣候這種長時間的氣象狀況及變化特徵，如季風或是較大範圍的氣壓變化，我覺得可以透過過去的氣象資料製作模型，甚至是訓練AI預測出氣候的趨勢～

**Comment by Hsucuchia** [comment]
人工智慧在天氣預報方面的應用確實提高了預報準確度，但是它並不能完全取代人類智慧在天氣預報上的地位。在預測天氣事件時，應該綜合考量人工智慧和人類的專業知識和經驗，這樣才能得到更準確的預報結果。此外，專業人士的監督和客觀評估也是非常重要的，這可以幫助確保預報結果的合理性。最後，ChatGPT 取代天氣預報員的說法是不正確的，人工智慧可以輔助預報員的工作，但是無法完全取代他們的角色。

**Comment by llh26** [comment]
我認為現在的天氣越來越說不準了，突然變天的情況也越來越多。人工智慧的確在某些層面上提升了天氣預報的準確度，但這也不代表它會百分之百的準確，因為天氣變化有太多因素需要去考量，再怎麼預測也可能會有那百分之一的意料之外。對我來說天氣預報一直都只是一個「參考」，我從小就養成隨身帶著雨傘的習慣，因為天氣真的說不準⋯。
